app/components/post_draft/send_action/index.tsx

```diff
@@ -67,15 +103,32 @@ function SendButton({
             disabled={disabled}
             onLongPress={showScheduledPostOptions}
         >
-            <View style={viewStyle}>
-                <CompassIcon
-                    name='send'
-                    size={24}
-                    color={buttonColor}
-                />
-            </View>
+            <Tooltip
```

Comment:
Note to reviewer - I tried adding tests to assert this was visible but couldn't make the test wait for `InteractionManager.runAfterInteractions`.

Reply:
Have you tried mocking the behavior?

Reply:
Thanks @enahum ! It works great!